packages/backend/src/controllers/authentication.ts

```diff
@@ -136,6 +147,7 @@ export const googlePassportStrategy: GoogleStrategy | undefined = !(
                   const user = await userService.loginWithOpenId(
                       openIdUser,
                       req.user,
+                      req.session.inviteCode,
```

Comment:
The session.inviteCode should be cleared in this handler not in success/failure.